Gaijinizer patch/Map215.txt

```diff
@@ -1919,19 +1968,19 @@ Yo, “Yotori” ......! ? What about a bird demon
 
 基本的に、こいつらは人間を『つがい』にする
 > CONTEXT: Map215/events/7/pages/0/48/Dialogue/other1
-Yes Yes. It is classified as Harpy or Saleen.
-Basically, they make humans "paired"#MTLed
+That's right. They're classified as a kind of
+harpy or siren, but all you need to know is
+their tendency to pair bond with humans.#Ron
 > END STRING
 
 > BEGIN STRING
 似たような奴で例えると、バーゲストみたいな感じだが
 
 妖鳥の特徴として『個体に対する執着心』の強さがある。
 > CONTEXT: Map215/events/7/pages/0/53/Dialogue/other1
-If you compare it with similar people, it feels
-like a barghest. There is a strength of
-"attachment to individuals" as a feature of the
-sorceress.#MTLed
+They're similar to Barghests in that regard. They
+have a strong propensity toward becoming attached
+to certain individuals they regard as special.#RonIffyWording
 > END STRING
```

Comment:
Minor wording/flow changes:
>_**They're actually similar to Barghests in that
 regard. They have a strong instinct towards
becoming attached to individuals they regard as
special.**_